package.json

```diff
@@ -301,7 +301,7 @@
     "@metamask/browser-passworder": "^4.3.0",
     "@metamask/contract-metadata": "^2.5.0",
     "@metamask/controller-utils": "^11.4.0",
-    "@metamask/design-tokens": "^5.0.0",
+    "@metamask/design-tokens": "npm:@metamask-previews/design-tokens@5.0.0-preview.3ab3ce4",
```

Comment:
This needs to be updated once the design tokens package has been published with brand evolution colors